pkg/util/kubernetes/apiserver/diagnosis.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package apiserver
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
+	log "github.com/cihub/seelog"
+	"github.com/ericchiang/k8s"
+)
+
+func init() {
+	diagnosis.Register("K8s API Server availability", new(apiServerAvailabilityDiagnosis))
```

Comment:
I added this but it's not registered for the agent because the `apiserver` package seems unused for now